tox.ini

```
@@ -27,6 +27,7 @@ commands =
 
 [testenv:docs]
 basepython = python3.9
+skip_install = true
 envdir = .tox/docs
 passenv = DOCS_FROM_MASTER QISKIT_DOCS_BUILD_TUTORIALS
```

```suggestion
passenv = DOCS_FROM_MASTER, QISKIT_DOCS_BUILD_TUTORIALS
```

Also see https://tox.wiki/en/latest/faq.html#tox-4-changed-ini-rules